test/spec/mobileAndAmpRender_spec.js

```diff
@@ -287,12 +287,12 @@ describe('writeAdHtml', () => {
   it('removes DOCTYPE from markup', () => {
     const ps = sinon.stub();
     writeAdHtml('<!DOCTYPE html><div>mock-ad</div>', ps);
-    sinon.assert.calledWith(ps, sinon.match.any, '<div>mock-ad</div>')
+    sinon.assert.calledWith(ps, '<div>mock-ad</div>')
```

Comment:
Could you add a test case for the regression? "if I writeAdHtml a sript, it should run".